app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/_module.less

```diff
@@ -293,6 +293,12 @@
         }
     }
 
+    .page-product-grouped {
```

Comment:
Grouped product specific code should be placed under right module Magento__GroupedProduct in this case.